mubench.pipeline/tests/tasks/implementations/test_publish_findings_task.py

```diff
@@ -49,15 +50,17 @@ def setup(self):
 
         self.uut = PublishFindingsTask(self.experiment_id, "/sources", "http://dummy.url", "-username-", "-password-")
 
-    def test_post_url(self, post_mock):
+    def test_post_url(self, post_mock, getsize_mock):
+        getsize_mock.return_value = 0
```

Comment:
Do we really need to do this in every test? Doesn't the patch use some sensible default? Or can it not, due to no type information? If so, can we move this to the setup somehow?